### PR TITLE
feat: add feature A

### PR DESCRIPTION
**Pull request by Calvin-Aziszam-S**
code added for feature A

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
+		<link rel="stylesheet" href="index.css" />
+		<title>Pertemuan 3</title>
+	</head>
+	<body>
+		<main>
+			<section>
+				<nav id="header">Meeting 3</nav>
+				<section id="content">
+					<div class="content-section">Content</div>
+					<div class="feature-a">Content for Feature A</div>
```
Comment (prakas26) [inline index.html:16]: tambah Feature B
<div class="feature-b">Content for Feature B</div>

**Comment by prakas26** [comment]
tambah Feature B

Content for Feature B

**Comment by Calvin-Aziszam-S** [comment]
> tambah Feature B
> 
> Content for Feature B

noted mas

File: index.html (hunk 1 above)
Comment (Calvin-Aziszam-S) [reply index.html:16]: oke, noted mas

File: index.html [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
+					<div class="feature-b">Content for Feature B</div>
```
Comment (Calvin-Aziszam-S) [inline index.html:17]: feature B has been added as per comment.

**Pull request merged**